src/jot.gleam

```diff
@@ -147,13 +148,73 @@ fn parse_document(
       }
     }
 
+    [":", ..in] -> {
+      case parse_div(in, attrs) {
+        None -> {
+          let #(paragrah, in) = parse_paragraph(in, attrs)
+          parse_document(in, refs, [paragrah, ..ast], dict.new())
+        }
+        Some(#(div, in)) -> {
+          parse_document(in, refs, [div, ..ast], dict.new())
+        }
+      }
+    }
+
     _ -> {
       let #(paragraph, in) = parse_paragraph(in, attrs)
       parse_document(in, refs, [paragraph, ..ast], dict.new())
     }
   }
 }
 
+fn parse_div(
+  in: Chars,
+  attrs: Dict(String, String),
+) -> Option(#(Container, Chars)) {
+  use #(class, count, in) <- option.then(parse_div_start(in, 1))
+  let #(content, in) = parse_div_content(in, count, "")
+  let d = parse(content)
+  Some(#(Div(attrs, class, d.content), in))
```

Comment:
i have not found a better way to accomplish this, but if there are any ideas i can try them out

Reply:
To do what, sorry?